packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -1040,6 +1105,12 @@ export class SamplerOperations {
                                 .map(poolAddress =>
                                     this.getBalancerSellQuotes(poolAddress, makerToken, takerToken, takerFillAmounts),
                                 );
+                        case ERC20BridgeSource.Cream:
+                            return this.creamPoolsCache
+                                .getCachedPoolAddressesForPair(takerToken, makerToken)!
+                                .map(poolAddress =>
+                                    this.getCreamSellQuotes(poolAddress, makerToken, takerToken, takerFillAmounts),
```

Comment:
nit: maybe we could just use `getBalancerSellQuotes` or nah? Don't know if we need `CreamFillData` when it's the same as `BalancerFillData`